frontend/test/pytest/test_transform.py

```diff
@@ -586,3 +1125,374 @@ def _id(tape: qml.tape.QuantumTape) -> (Sequence[qml.tape.QuantumTape], Callable
             @qml.qnode(qml.device(backend, wires=1))
             def f():
                 return qml.state()
+
+
+@pytest.mark.xfail(reason="Noise models not supported on Lightning.")
+def test_add_noise(backend):
```

Comment:
If you use a non-channel noise, this might work (very rough example based on the one you use here):

```py
import pennylane as qml
import numpy as np
from functools import partial
import jax
from catalyst import measure, qjit

device_name = "lightning.qubit"

fcond1 = qml.noise.op_eq(qml.RX) & qml.noise.wires_in([0, 1])
noise1 = qml.noise.partial_wires(qml.RX, 0.4)

fcond2 = qml.noise.op_in([qml.RX, qml.RZ])
def noise2(op, **kwargs):
    qml.CRX(op.data[0], wires=[op.wires[0], (op.wires[0] + 1) % 2])

noise_model = qml.NoiseModel({fcond1: noise1, fcond2: noise2})

@partial(qml.transforms.add_noise, noise_model=noise_model)
@qml.qnode(qml.device(device_name, wires=2), interface="jax")
def qfunc(w, x, y, z):
    qml.RX(w, wires=0)
    qml.RY(x, wires=1)
    qml.CNOT(wires=[0, 1])
    qml.RY(y, wires=0)
    qml.RX(z, wires=1)
    return qml.expval(qml.Z(0) @ qml.Z(1))

expected = jax.jit(qfunc)(0.9, 0.4, 0.5, 0.6)
observed = qjit(qfunc)(0.9, 0.4, 0.5, 0.6)
print(expected, observed) # 0.6365939421040764 0.6365939421040762
```

Reply:
Thanks for this! I updated the test.